lib/phoenix_mtm/changeset.ex

```diff
@@ -41,6 +41,7 @@ defmodule PhoenixMTM.Changeset do
   end
 
   defp all(ids, repo, mod) do
-    repo.all(from m in mod, where: m.id in ^ids)
+    ids_as_ints = Enum.map(ids, fn(i) -> if String.length(i) > 0, do: String.to_integer(i) end)
```

Comment:
I was getting a error because there were strings instead of integers from the form params.

<pre>
** (Ecto.Query.CompileError) `r.id() in ["2", "1", ""] | Repo.all()` is not a valid query expression.
</pre>